devimint/src/tests.rs

```diff
@@ -1915,6 +1928,10 @@ pub async fn test_client_config_change_detection(
     let config_dir = PathBuf::from(&data_dir);
 
     // Shutdown all federation servers
+    //
+    // In prod. one would probably use a coordinated shutdown, just to be
+    // careful, but since the change is only adding a new module, there is
+    // no way for the consensus to diverge.
```

Comment:
Even adding a module can make consensus diverge afaik if it sends CIs right after being activated and some peers haven't added it yet.